### ISSUE TITLE
Create a high-level walk-through of the tutorial

### ISSUE THREAD

**Issue by ethanfrey** [iov-one/tutorial#15]
"best practices" now contains quite a bit of useful info, and the PRs are organized such that the diffs can be used to understand the system one component at a time.

These docs can either be in github markdown, readthedocs, a wiki, or other format which can easily be read by anyone with a browser. We should consider the format which is easiest for us to collectively maintain.

The end-goal should be something vaguely like https://cosmos.network/docs/tutorial/ to guide someone through building an app. We should link to weave.rtfd.io for all the theory sections, and keep this focused on the code samples and usage of the API.

Please create multiple sub-issues to track progress on this, once the tutorial format is designed. All general decisions (like how, what, audience) should go here.

**Comment by orkunkl** [comment]
I think keeping tutorial docs in readthedocs format would be better. Because at the end we will have to convert all formats to *rst* format

**Comment by ethanfrey** [comment]
I agree. Let's find a good solution to standardize around with https://github.com/iov-one/weave/issues/726 and then do that... not yet another apporach.
